source/ts/app-connection.ts

```diff
@@ -87,6 +87,10 @@ export class AppConnection extends EventEmitter {
       env,
     });
 
+    this.process?.on('error', (error) => {
```

Comment:
Do you need `?` here? I think it should be assigned...

Reply:
thanks, have amended https://github.com/FocusriteGroup/juce-end-to-end/pull/227/commits/0c8b09fdf87161f4e9126dcc751766a5e45af792